--- nano-1.2.1/nano.h~	Sun May 11 16:35:14 2003
+++ nano-1.2.1/nano.h	Sun May 11 16:35:14 2003
@@ -50,6 +50,8 @@
 #define KEY_DC SL_KEY_DELETE
 #elif defined(HAVE_NCURSES_H)
 #include <ncurses.h>
+#elif defined(HAVE_NCURSES_NCURSES_H)
+#include <ncurses/ncurses.h>
 #else /* Uh oh */
 #include <curses.h> 
 #endif /* CURSES_H */
--- nano-1.2.1/configure.ac~	Sun May 11 16:36:24 2003
+++ nano-1.2.1/configure.ac	Sun May 11 16:36:24 2003
@@ -295,7 +295,7 @@
 
 if eval "test x$CURSES_LIB_NAME = x"
 then
-    AC_CHECK_HEADERS(curses.h ncurses.h)
+    AC_CHECK_HEADERS(curses.h ncurses.h ncurses/ncurses.h)
     AC_CHECK_LIB(ncurses, tgetent, [CURSES_LIB="-lncurses" CURSES_LIB_NAME=ncurses])
 fi
 
