--- ./nano-1.9.99pre2/src/nano.h.orig	2006-08-22 20:48:13.000000000 +0200
+++ ./nano-1.9.99pre2/src/nano.h	2006-09-14 00:10:39.000000000 +0200
@@ -71,6 +71,8 @@
 /* Ncurses support. */
 #elif defined(HAVE_NCURSES_H)
 #include <ncurses.h>
+#elif defined(HAVE_NCURSES_NCURSES_H)
+#include <ncurses/ncurses.h>
 #else
 /* Curses support. */
 #include <curses.h>
--- ./nano-1.9.99pre2/configure.ac.orig	2006-08-29 19:29:57.000000000 +0200
+++ ./nano-1.9.99pre2/configure.ac	2006-09-14 00:10:31.000000000 +0200
@@ -400,7 +400,7 @@
 dnl Checks for libraries.
 
 if eval "test x$CURSES_LIB_NAME = x"; then
-    AC_CHECK_HEADERS(ncurses.h)
+    AC_CHECK_HEADERS(ncurses.h ncurses/ncurses.h)
 
     if test x$enable_utf8 != xno; then
 	AC_CHECK_LIB(ncursesw, get_wch, [CURSES_LIB="-lncursesw" CURSES_LIB_NAME=ncursesw CURSES_LIB_WIDE=yes])
