--- nano-1.1.12/Makefile.am~	Wed Oct 30 07:33:13 2002
+++ nano-1.1.12/Makefile.am	Wed Oct 30 07:33:13 2002
@@ -18,7 +18,7 @@
 		winio.c 
 
 man_MANS = nano.1
-nano_LDADD = @GLIB_LIBS@ @LIBINTL@
+nano_LDADD = $(GLIB_LIBS) $(LIBINTL)
 
 info_TEXINFOS = nano.texi
 MAKEINFO = makeinfo --no-split
@@ -31,6 +31,6 @@
 SUBDIRS = po intl
 
 localedir = $(datadir)/locale
-INCLUDES = -Iintl -DLOCALEDIR=\"$(localedir)\" -I@includedir@
+INCLUDES = -Iintl -DLOCALEDIR=\"$(localedir)\"
 
 ACLOCAL_AMFLAGS = -I m4
