--- nano-1.1.10/Makefile.am.orig	Mon Jun  3 14:32:28 2002
+++ nano-1.1.10/Makefile.am	Wed Aug  7 06:29:31 2002
@@ -28,7 +28,7 @@
 		mkinstalldirs nano.1 nano.1.html faq.html nanorc.sample \
 		nano.spec
 
-SUBDIRS = po m4
+SUBDIRS = po intl
 
 localedir = $(datadir)/locale
 INCLUDES = -Iintl -DLOCALEDIR=\"$(localedir)\" -I@includedir@
--- nano-1.1.12/configure.ac.orig	Fri Oct 25 04:58:33 2002
+++ nano-1.1.12/configure.ac	Wed Oct 30 06:51:02 2002
@@ -4,9 +4,7 @@
 AC_CONFIG_SRCDIR([nano.c])
 AC_PREREQ(2.52)
 AM_INIT_AUTOMAKE(nano, 1.1.12)
-AM_CONFIG_HEADER(config.h:config.h.in)
-dnl AM_ACLOCAL_INCLUDE(m4)
-ALL_LINGUAS="es de fr it id fi hu ca cs gl uk ru sv nn nl da pl nb ms pt_BR"
+AM_CONFIG_HEADER(config.h)
 
 dnl Checks for programs.
 AC_PROG_CC
@@ -313,7 +311,8 @@
 fi
 
 dnl i18n stuff - pretty incomplete for now
-AM_GNU_GETTEXT([external])
+ALL_LINGUAS="es de fr it id fi hu ca cs gl uk ru sv nn nl da pl nb ms pt_BR"
+AM_GNU_GETTEXT
 
-AC_CONFIG_FILES([Makefile m4/Makefile po/Makefile.in nano.spec])
+AC_CONFIG_FILES([Makefile m4/Makefile intl/Makefile po/Makefile.in nano.spec])
 AC_OUTPUT
