--- nano-2.2.5/configure.ac~	2010-08-09 09:26:51.987128085 +0200
+++ nano-2.2.5/configure.ac	2010-08-09 09:28:32.852952219 +0200
@@ -442,7 +442,7 @@
     AC_CHECK_HEADERS(ncurses.h ncurses/ncurses.h)
 
     if test x$enable_utf8 != xno; then
-	AC_CHECK_LIB(ncursesw, get_wch, [CURSES_LIB="-lncursesw" CPPFLAGS="-I/usr/include/ncursesw $CPPFLAGS" CURSES_LIB_NAME=ncursesw CURSES_LIB_WIDE=yes])
+	AC_CHECK_LIB(ncursesw, get_wch, [CURSES_LIB="-lncursesw -ltinfow" CPPFLAGS="-I/usr/include/ncursesw $CPPFLAGS" CURSES_LIB_NAME=ncursesw CURSES_LIB_WIDE=yes])
     fi
 
     if eval "test x$CURSES_LIB_NAME = x"; then
